CHANGELOG.rst

```diff
@@ -1,5 +1,31 @@
 .. _changelog:
 
+0.32.0
+------
+
+Renku ``0.32.0`` introduces improvements in the KG services, thus enhancing KG overall performance. 
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+- 🔁 **KG**: ``updatedAt`` renamed to ``dateModified`` on the Project Details API (`#1582 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1582>`_).
```

Comment:
Not sure that this is a user-facing change?

Reply:
Depends on the user. From the renku API user point of view, it could be viewed as that.